src/commands/user/export-mapping.ts

```diff
@@ -0,0 +1,42 @@
+import fs from 'fs'
+import path from 'path'
+
+import { flags } from '@oclif/command'
+import { Kommand } from '../../common'
+import { kuzzleFlags } from '../../support/kuzzle'
+
+export default class UserExportMapping extends Kommand {
+  static description = 'Exports user mapping to JSON.'
+
+  static flags = {
+    help: flags.help({}),
+    path: flags.string({
+      description: 'Dump directory',
+      default: 'userMapping'
```

Comment:
Should be the same as users data (collection data and mappings are dump in the same folder)
```suggestion
      default: 'users'
```